tea-config-generator/serverless.yml

```diff
@@ -24,16 +24,29 @@ functions:
   capabilities:
     handler: capabilities.capabilities
     events:
-      - http: GET ${self:custom.endPoint}/
-      - http: GET ${self:custom.endPoint}/capabilities
+      - alb:
+          listenerArn: ${self:custom.loadBalancer}
```

Comment:
This listernerArn can be defined as reference to CF outputs, i.e. `${cf:${opt:stage}.servicesLbListenerArn}` where stage is passed in as input to serverless deploy.

Reply:
is this present by default? I don't need to do anything to support it?

Reply:
fixed it.